site/source/docs/tools_reference/settings_reference.rst

```diff
@@ -151,6 +151,14 @@ Note that this setting does not affect the behavior of operator new in C++.
 This function will always abort on allocation failure if exceptions are disabled.
 If you want new to return 0 on failure, use it with std::nothrow.
 
+.. _initial_heap:
+
+INITIAL_HEAP
+============
+
+The initial amount of heap memory to use. This value is appended to the
```

Comment:
How about once more sentence between these two to make it very clear what "heap" means in this context..

e.g.

```
The initial amount of heap memory available to the program.  This the memory region available for dynamic allocations via `sbrk`, `malloc` and `new`.
```